be/src/exec/tablet_info.cpp

```diff
@@ -206,6 +220,8 @@ Status OlapTablePartitionParam::init(RuntimeState* state) {
         } else {
             _partitions_map.emplace(&part->end_key, part);
         }
+        VLOG(1) << "add partition:" << part->id << " start " << part->start_key.debug_string() << " end "
```

Comment:
This is a debug log, do we need to keep it?

Reply:
there are no side affect when we disable verbose log but useful for debug since there are not enough stable for automatic partition.